spec/partials/line-break-grammar-spec.coffee

```diff
@@ -18,17 +18,16 @@ describe 'Should tokenize line break when', ->
 
   it 'contains simple character', ->
     tokens = grammar.tokenizeLines '''
-       +
       Foo +
+      Bar
```

Comment:
My example come from AsciiDoctor code https://github.com/asciidoctor/asciidoctor/blob/master/lib/asciidoctor.rb#L970-L983

Are you sure of your change ?

I found a mistake in the AsciiDoctor code comment :
https://github.com/asciidoctor/asciidoctor/blob/master/lib/asciidoctor.rb#L664-L678

Roman letters cannot be follow by a `.` but by a `)` :wink: